PubMed.js

```diff
@@ -176,6 +179,16 @@ function getSearchResults(doc, checkOnly) {
 			checked: checkbox && checkbox.checked
 		};	
 	}
+
+	for (var i=0; i<docsums.length; i++) {
```

Comment:
should probably check whether `docsums` are found as on PubMed page they will not be